birkof/netopia-mobilpay-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+    "bundles": {
+        "birkof\\NetopiaMobilPay\\NetopiaMobilPayBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "gitignore": [
+        "/%CONFIG_DIR%/netopia_mobilpay/*.key",
+        "/%CONFIG_DIR%/netopia_mobilpay/*.cer"
```

Comment:
Good!

You can remove your `.gitignore` now.

Reply:
Done!